### ISSUE TITLE
2.1.2 No Keyboard Trap (A)

### ISSUE THREAD

**Issue by Kelvyn-Meyers** [UNLV-FIA/UNLV-FIA-Group#55]
### 2.1.2 No Keyboard Trap - Level A
If keyboard focus can be moved to a component of the page using a keyboard interface, then focus can be moved away from that component using only a keyboard interface, and, if it requires more than unmodified arrow or tab keys or other standard exit methods, the user is advised of the method for moving focus away. (Level A)

_Note:_ Since any content that does not meet this success criterion can interfere with a user's ability to use the whole page, all content on the Web page (whether it is used to meet other success criteria or not) must meet this success criterion. See [Conformance Requirement 5: Non-Interference](https://www.w3.org/TR/WCAG20/#cc5).

[Understanding 2.1.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-trapping.html)
[How to Meet 2.1.2](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=131%2C133%2C211#no-keyboard-trap)
[How to Test 2.1.2](https://usfs.box.com/s/knn5w8jul8g0xicrnpa4cxzk07t8rc1d)

**Trusted Tester ID:**
- [x] Test 4.C no-keyboard-trap

**Accessibility Principle:** Operable (Keyboard Accessible)
